src/AspNetCoreModuleV2/AspNetCore/Inc/applicationmanager.h

```diff
@@ -65,7 +65,7 @@ class APPLICATION_MANAGER
     HRESULT
     GetOrCreateApplicationInfo(
```

Comment:
we can pass ihttpcontext here. It makes sense to get the application for a request. Can be done in next PR